docs/containers/kubernetes-env-vars.md

```diff
@@ -0,0 +1,96 @@
+---
+Area: containers
+ContentId: 8c1ee589-25bb-4c2d-ab8d-7ccfa7b4e282
+PageTitle: Use Kubernetes service environment variables for service to service communication
+DateApproved: 02/12/2021
+MetaDescription: Learn how to use Kubernetes service environment variables with Bridge to Kubernetes to enable service-to-service communication as a normal user in a Kubernetes cluster
+---
+# Use Kubernetes service environment variables
+
+When you communicate with another service in the same Kubernetes cluster, for example with an HTTP request, you typically use the service name in the URL for the request. Bridge to Kubernetes reroutes traffic by modifying the host name resolution to redirect network traffic to its own version of the services. This works in most scenarios, but fails in the case where the Bridge to Kubernetes process has restricted privilege, such as when the request originates from a normal user account or when using VS Code remoting through SSH. This is because in order to enable the name resolution for redirected services, Bridge to Kubernetes needs to modify the hosts file, but that is not possible when Bridge to Kubernetes runs from a normal user account. To work around this issue, you can write your code to use the Kubernetes service environment variables instead of a hardcoded service name.
+
+## Kubernetes service environment variables
+
+The following table shows the Kubernetes service environment variables that are available from any service in the cluster, for an example service using the TCP protocol on a port. The *servicename* is the name of the service, converted to uppercase, and with hyphens converted to underscores, so for example, a service named webapp-2 yields an environment variable named WEBAPP_2_SERVICE_HOST.
```

Comment:
here, also instead of webapp-2, can we just use the name web-api?

Reply:
Yes, that makes sense